<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_action</class>
 <widget class="QDialog" name="add_action">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_action">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Доходы</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Расходы</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_category">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_money">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
